dash/examples/bench.07.local-copy/main.cpp

```diff
@@ -22,6 +26,16 @@ using std::endl;
 using std::setw;
 using std::setprecision;
 
+#ifndef __APPLE__
+  #define _aligned_malloc(size, alignment) memalign(alignment, size)
```

Comment:
why not use `posix_memalign` instead of `memalign` alltogether? (`memalign` is deprecated anyway)